modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test/src/testIntegration/java/com/liferay/site/initializer/extender/internal/test/BundleSiteInitializerTest.java

```diff
@@ -4175,6 +4175,35 @@ private void _assertUserAccounts1() throws Exception {
 		Assert.assertTrue(userAccount.getImageId() == 0);
 
 		_assertUserSiteGroups(userAccount.getId());
+
+		userAccount = userAccountResource.getUserAccountByEmailAddress(
+			"test.user3@liferay.com");
+
+		Assert.assertNotNull(userAccount);
+
+		accountBriefs = userAccount.getAccountBriefs();
+
+		Assert.assertEquals(
+			Arrays.toString(accountBriefs), 0, accountBriefs.length);
+
+		Assert.assertEquals(
+			"testalternatename3", userAccount.getAlternateName());
+		Assert.assertEquals(
+			UserAccount.Status.INACTIVE, userAccount.getStatus());
+
+		organizationBriefs = userAccount.getOrganizationBriefs();
+
+		Assert.assertEquals(
+			Arrays.toString(organizationBriefs), 0, organizationBriefs.length);
+
+		userGroupBriefs = userAccount.getUserGroupBriefs();
+
+		Assert.assertEquals(
+			Arrays.toString(userGroupBriefs), 0, userGroupBriefs.length);
+
+		Assert.assertTrue(userAccount.getImageId() == 0);
+
+		_assertUserSiteGroups(userAccount.getId());
```

Comment:
I think these can be removed as we are just testing that we are creating the user without accounts (all the other things should be already verified by previous test data)